packages/ra-ui-materialui/src/form/FormTabHeader.tsx

```diff
@@ -44,6 +44,12 @@ export const FormTabHeader = ({
     );
 };
 
+const UseFormStateOptions = {
```

Comment:
Why not calling this only `FormStateOptions`? Is it by convention?

Reply:
Because that's the options of the `useFormState` hook. No convention